pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, greenlet
+, websockets
+, pyee
+, pillow
+, pixelmatch
+, autobahn
+, pyopenssl
+, service-identity
+, requests
+, objgraph
+, callPackage
+}:
+
+buildPythonPackage rec {
+  pname = "playwright";
+  version = "1.21.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "playwright-python";
+    rev = "v${version}";
+    sha256 = "sha256-+q95w1UMCEZeYeXvlLagixUAROakdJFrIBy1rPMyYZc=";
+  };
+
+  format = "pyproject";
+
+  playwright-driver = callPackage ./driver.nix { playwrightVersion = version; };
+
+  postPatch = ''
+    # works with pyee 9 as well, which is what nixos has packaged
+    # disables setuptools-scm, as it fails
+    substituteInPlace meta.yaml --replace "- pyee ==8.1.0" ""
+    substituteInPlace setup.py \
+        --replace '"pyee==8.1.0",' "" \
+        --replace '"setuptools-scm==6.3.2",' ""
+    # replace functionality that setuptools-scm provides by adding the version to a file
+    echo 'version = "${version}"' > playwright/_repo_version.py
+  '';
+
+  preBuild = ''
+    mkdir -p driver
+    cp -r ${playwright-driver}/driver ./
+  '';
+
+  preInstall = ''
+    # As pip names the wheel "dist", but when installing it will expect a properly formatted filename in dist
+    mv dist dist.orig
+    mkdir -p dist
+    mv dist.orig dist/playwright-${version}-py3-none-any.whl
+  '';
+
+  propagatedBuildInputs = [
+      playwright-driver
+      greenlet
+      websockets
+      pyee
+      pillow
+      pixelmatch
+      autobahn
+      pyopenssl
+      service-identity
+      requests
+      objgraph
```

Comment:
What's up with the indent here? Please run `nixpkgs-fmt`

Reply:
Updated.